notebooks/community/prediction/custom_prediction_routines/SDK_Custom_Preprocess.ipynb

```diff
@@ -0,0 +1,1744 @@
+{
```

Comment:
<p>Add what to expect in logs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='92'/>